pilot/database/database.py

```diff
@@ -132,7 +132,7 @@ def get_last_development_step(app_id, last_step=None):
     last_dev_step = last_dev_step_query.order_by(DevelopmentSteps.id.desc()).first()
 
     # If a step is found, convert it to a dictionary, otherwise return None
-    return model_to_dict(last_dev_step) if last_dev_step else None
+    return model_to_dict(last_dev_step, recurse=False) if last_dev_step else None
```

Comment:
This function (`get_last_development_step`) is only ever used in `get_created_apps_with_steps`.